### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by anilbey**
Inspired by: https://github.com/BlueBrain/luigi-tools/blob/main/.github/dependabot.yml

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```
Comment (AurelienJaquier) [inline .github/dependabot.yml:8]: Shouldn't you use "github-actions" here, like in the luigi-tools yml and like in the [documentation](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#example-dependabotyml-file-for-github-actions)?

Comment (anilbey) [reply .github/dependabot.yml:8]: Ah yes, I don't know why the default generated yml did not set it. Thanks, let me add it.

**Review by AurelienJaquier — APPROVED** [review]
looks good to me! :)

**Review by ilkilic — APPROVED** [review]
Nice!

**Comment by anilbey** [comment]
Great, I'm merging.

**Pull request merged**